### DISCUSSION TITLE
Remove name squatting policy

### DISCUSSION THREAD

**Discussion by kamikazechaser** [community/community#37362] [Other Feature Feedback, Questions, & Ideas]
According to the docs [here](https://docs.github.com/en/site-policy/other-site-policies/github-username-policy#name-squatting-policy) Name-squatting is prohibited. While, according to an official response on #32361, Github won't enforce the policy.

I suggest removing the policy completely so as to not give false hope and create confusion. And if there is a reason to keep the policy, it should be publicly mentioned.

**Comment by ossanna16** [comment]
Hi there @kamikazechaser and welcome to our community! Thank you for the suggestion 🙂

**Comment by landsman** [comment]
Yeah, exactly. No longer supported. Please do something about that - support it in defined process 🙏 

https://docs.github.com/en/site-policy/other-site-policies/github-username-policy#name-squatting-policy
https://support.github.com/contact/account

![Screenshot 2023-08-08 at 10 40 13](https://github.com/community/community/assets/3184228/477d85d5-de63-4531-84b5-2e460016bdac)

**Comment by terabytesoftw** [comment]
Yes, it should be eliminated, there is no point in maintaining a policy that is not followed.
